package.json

```diff
@@ -11,7 +11,8 @@
     "test-watch": "jest --coverage --watch",
     "cypress:open": "cypress open",
     "cypress:cli": "cypress run",
-    "cypress:travis": "http-server dist -c-1 -s & cypress run"
+    "cypress:travis": "http-server dist -c-1 -s & cypress run",
+    "pa11y-run": "http-server dist -c-1 -s & ./node_modules/.bin/pa11y-ci"
```

Comment:
Please add a `prepa11y-run` script which runs `npm run-script build`, to ensure that the a11y tests are always run on a fresh build of the website.

See NPM's `package.json` docs for more info: https://docs.npmjs.com/misc/scripts

Reply:
and update the docs to reflect this change too!